src/main/java/org/zalando/fahrschein/EventAlreadyProcessedException.java

```diff
@@ -0,0 +1,15 @@
+package org.zalando.fahrschein;
+
+public class EventAlreadyProcessedException extends Exception {
```

Comment:
> Why did you decided to go with checked exceptions?

So that I can't forget to handle it in `NakadiReader`. It's part of the interface between the `Listener` and `NakadiReader`.